pages/components/Alerts/alerts.html

```diff
@@ -76,7 +76,7 @@
                         <a href="/pages/components/Typography/typography.html">Typography</a>
                     </li>
                     <li class="sidebar-list-item">
-                        <a href="#">Toast</a>
```

Comment:
you could have made whole li a link because now I have to specifically hover over the text for navigating